src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,74 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+import duke.DukeException;
+
+/**
+ * Deadline is a subtype of Task.
+ * An event is meant to be for when a task should be done before a certain date.
+ * Eg. Return book 19/01/2020
+ *
+ * @author Dargo
+ */
+public class Deadline extends Task {
+
+    private String deadlineDateString;
```

Comment:
You do you need to store the string format of the date as well? Instead, you can have a static method(or multiple methods) which will convert LocalDate to string as necessary

Reply:
Resolved by adding a new method in Task.java to format date and format time.